kolibri/plugins/setup_wizard/middleware.py

```diff
@@ -10,6 +12,16 @@
     "session-list"
 ]
 
+SETUP_WIZARD_URL = reverse("kolibri:setupwizardplugin:setupwizard")
+
+redirect_page_content = """
+<html xmlns="http://www.w3.org/1999/xhtml">
+  <head>
+    <meta http-equiv="refresh" content="0;URL='{url}'" />
+  </head>
+</html>
+""".format(url=SETUP_WIZARD_URL)
```

Comment:
looks good! might want to just add a comment like:

```python
# Temporary workaround for https://github.com/learningequality/kolibri/issues/3852
# Uses a meta tag redirect rather than a 302 to prevent extra load by Windows GUI polling

Reply:
Done, though I didn't put "temporary", as this is reasonable to keep as-is later, even after we fix those issues in other ways as well.